src/OVAL/oval_generator.c

```diff
@@ -114,6 +117,16 @@ char *oval_generator_get_timestamp(struct oval_generator *generator)
 	return generator->timestamp;
 }
 
+const char *oval_generator_get_platform_schema_version (struct oval_generator *generator, const char *platform)
+{
+	char *pschv = oscap_htable_get(generator->platform_schema_versions, platform);
```

Comment:
I personally prefer "char* platform_schema_version". It is long, but I see what does it mean. I need to see "oscap_htable_get(generator->platform_schema_versions, platform);" to understand pschv. But is is short fragment of code, maybe its ok.